source/class/cv/plugins/Clock.js

```diff
@@ -35,6 +34,59 @@ qx.Class.define('cv.plugins.Clock', {
     this.base(arguments, props);
   },
 
+  /*
+  ******************************************************
+  PROPERTIES
+  ******************************************************
+  */
+  properties: {
+    src: {
+      check: 'String'
+    },
+    src_popup: {
+      check: 'String'
+    },
+    title_popup: {
```

Comment:
We should stick to CamelCase property names, the parser can map them accordingly. IMHO the underscore names doesn't really fit in when everything else is CamelCased.